test-ciphers.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+console_url=$(oc get console.config.openshift.io cluster --template '{{.status.consoleURL}}')
+if [ -z "${console_url}" ]
+then
+  echo "no console_url, are you connected to a cluster?"
+  exit 1
+fi
+
+# need to format the console_url for s_client
+console_url_without_http=${console_url#"https://"}
+console_url_without_http_with_port="${console_url_without_http}:443"
+SERVER="${console_url_without_http_with_port}"
+
+
+# ensure what we expect works
+# we approve roughly 16 cipher suites
+# but
+valid_cipher_sample=(
+  ECDHE-ECDSA-CHACHA20-POLY1305
+  ECDHE-RSA-AES256-GCM-SHA384
+)
+
+echo "testing valid ciphers...."
+# we want these to successfully connect
+for CIPHER in "${valid_cipher_sample[@]}"
```

Comment:
I'd stick to consistent variable naming. Sometimes you use all caps, sometimes you use snake case

Reply:
Yeah, I should have fixed that.  I tinkered with this in several different ways and got a bit copy-pasty between them for my final pass (yuk).